test/libsyncengine/reconciliation/platform_inconsistency_checker/testplatforminconsistencycheckerworker.cpp

```diff
@@ -42,16 +43,26 @@ namespace KDC {
 #define MAX_NAME_LENGTH_WIN_SHORT 255
 
 void TestPlatformInconsistencyCheckerWorker::setUp() {
-    // Create SyncPal
+    // Create parmsDb
     bool alreadyExists = false;
     std::filesystem::path parmsDbPath = Db::makeDbName(alreadyExists, true);
-    std::filesystem::remove(parmsDbPath);
     ParmsDb::instance(parmsDbPath, "3.4.0", true, true);
-    ParametersCache::instance()->parameters().setExtendedLog(true);
 
-    SyncPath syncDbPath = Db::makeDbName(1, 1, 1, 1, alreadyExists);
-    std::filesystem::remove(syncDbPath);
-    _syncPal = std::make_shared<SyncPal>(syncDbPath, "3.4.0", true);
+    // Insert user, account, drive & sync
+    User user(1, 1, "dummy");
+    ParmsDb::instance()->insertUser(user);
+
+    Account account(1, 1, user.dbId());
+    ParmsDb::instance()->insertAccount(account);
+
+    Drive drive(1, 1, account.dbId(), std::string(), 0, std::string());
+    ParmsDb::instance()->insertDrive(drive);
+
+    Sync sync(1, drive.dbId(), _tempDir.path, "");
+    ParmsDb::instance()->insertSync(sync);
+
+    // Create SyncPal
+    _syncPal = std::make_shared<SyncPal>(sync.dbId(), "3.4.0");
     _syncPal->_syncDb->setAutoDelete(true);
     _syncPal->_tmpBlacklistManager = std::make_shared<TmpBlacklistManager>(_syncPal);
```

Comment:
Is `_tmpBlacklistManager ` used by the tests? If not, you may instantiate a `TestSync` object instead. (The manager could also have its own unit tests).

Reply:
Yes, it is used by